### ISSUE TITLE
Custom styling, bar background positioning

### ISSUE THREAD

**Issue by thinkMichel** [lokku/jquery-nstslider#19]
Given the following customization:
![image](https://cloud.githubusercontent.com/assets/813869/6885941/0cd08992-d62c-11e4-9bbf-e5c8fceb82cc.png)

I had to add some CSS tricks to get the gray '.bar' aligned correctly on the left side. The rounded corners were starting at the center of the left handle, and not the beginning. On the right side it goes all the way to the right border of the right grip correctly. Zoomed image:

![image](https://cloud.githubusercontent.com/assets/813869/6885954/2d5f0e1c-d62d-11e4-8069-aa76788d6890.png)

i made the grips sem-transparent to see whats going on. The same thing happens with the CSS you supply with the plugin, if you hide the handles, you can see that the bar does not start at the left side.

I fixed it with CSS for now but would be better if i knew where in the code this goes wrong.

**Comment by 11digitlabs** [comment]
@thinkMichel I have the same issue, the bar does not start all the way to the left.

![basicslider](https://cloud.githubusercontent.com/assets/2484390/8379443/ccc6e9b6-1c28-11e5-90b4-f13f161cae44.png)


What CSS trickery did you use to solve this? Thank you in advance!

**Comment by darksmo** [comment]
Hi @thinkMichel  and @11digitlabs.This should be fixed in the latest version v1.0.12. You can already see it in action on the demo page: lokku.github.io/jquery-nstslider/.

**Comment by 11digitlabs** [comment]
@darksmo thanks a lot! Works like a charm!

**Comment by 11digitlabs** [comment]
@darksmo apparently not after all :(
I have two different sliders.

The problem is fixed in the slider that starts in the middle of the slider.

The other one that starts all the way from the left, still has some inline `left` style.
![screen shot 2015-06-30 at 10 10 43](https://cloud.githubusercontent.com/assets/2484390/8425632/6350635a-1f10-11e5-96a6-cc483086660b.png)

Thank you!

**Comment by thinkMichel** [comment]
Hi, I havent tried the latest version. But i fixed it by applying a negative left-margin to the .bar class, with an equal left padding to balance it out again. The size of the margin depends on the size of the handle. 

In my case it was:

```css
.price-slider .bar {
  padding-left: 7px;
  margin-left: -7px;
```

Although you should experiment in your own code, because i have heavily styled the slider and consequently changed the css a lot. I Wouldn't mind posting the css, but i fear it wont be of any use to you..

**Comment by darksmo** [comment]
@11digitlabs I don't see it on http://lokku.github.io/jquery-nstslider/ (it runs the latest version). Maybe you could make a jsfiddle or adapt the demo/index.html page in this repository so that I can reproduce? Also make sure you are running v1.0.12 (sounds like it's not)

**Comment by 11digitlabs** [comment]
@darksmo I made a JS fiddle to show you the issue

http://jsfiddle.net/28YCS/36/

I'm definitely using the 1.10.12 as you can see in the fiddle. Drag the slider to the right so you can see the background color changing. You will see that there is a left:5px on the bar (the number depends on the width of the slider, so if its wider, the left is higher)

![screen shot 2015-07-14 at 09 49 27](https://cloud.githubusercontent.com/assets/2484390/8667857/aa44ad86-2a0d-11e5-90e4-b1696cc772b7.png)

Thanks!

**Comment by thinkMichel** [comment]
Looks like the same issue i had. Didnt have the time back then to search through the code and fix it (and possibly break something else). 

You could use my quick fix to solve your problem right now by adding the negative margin. In your fiddle if i set the .bar with a left margin of -5px at least it looks good for that size.

I had only 4 sizes to worry about at the time so  it was easy for me.

**Comment by 11digitlabs** [comment]
@thinkMichel Thank you for the suggestion. @darksmo said that he has fixed this issue, so I'll wait a bit for his reply. If not, i ll go with the css solution you proposed

**Comment by darksmo** [comment]
@11digitlabs Thanks for uploading the jsfiddle. For me it was really hard to understand that this bug occurs in the one handle setting only, with the bar set to the max or min value. From the screenshot the bit with the red arrow looked like a white handle to me! So it's now fixed in v1.0.13.
